tools/gromacs/rmsd.xml

```diff
@@ -0,0 +1,150 @@
+<tool id="gmx_rmsd" name="GROMACS RMSD calculation" version="@TOOL_VERSION@+galaxy@GALAXY_VERSION@">
+    <description>of molecular structures</description>
+    <macros>
+        <import>macros.xml</import>
+        <token name="@GALAXY_VERSION@">0</token>
+    </macros>
+    <expand macro="requirements"/>
+    <command detect_errors="exit_code"><![CDATA[
+
+    ln -s '$traj_input' ./traj_input.${traj_input.ext} &&
+    ln -s '$structure_input' ./structure_input.${structure_input.ext} &&
+    #if '$traj2.traj2select' == 'Yes':
+    ln -s '$traj2.traj_input2' ./traj_input2.${traj_input2.ext} &&
+    #end if
+    #if '$ndx_input':
+    ln -s '$ndx_input' ./ndx_input.${ndx_input.ext} &&
+    #end if
+    #if '$fit.fit_type' == 'none':
+    echo '$index_rmsdgroup' | gmx rms
+    #else
+    echo '$fit.index_refgroup' '$index_rmsdgroup' | gmx rms
+    #end if
+    ##inputs
+    -f ./traj_input.${traj_input.ext}  -s ./structure_input.${structure_input.ext}
+    #if '$traj2.traj2select' == 'Yes':
+    -f2 ./traj_input2.${traj_input2.ext}
+    #end if
+    #if '$ndx_input':
+    -n ./ndx_input.${ndx_input.ext}
+    #end if
+
+    -fit '$fit.fit_type'
+    -xvg '$outputformat'
+    ## other options
+    #if $mir == 'true':
+    -mir
+    #end if
+
+    >> verbose.txt 2>&1
+ 
+    ]]></command>
+    <inputs>
+        <param name="traj_input" type="data" format="trr,xtc" label="Trajectory file" help="In XTC or TRR format"/>
+        <conditional name="traj2">
+            <param name="traj2select" type="select" label="Add an additional trajectory file?" help="Can be useful when compraring two different simulations of the same system.">
```

Comment:
So my understanding is that it generates a correlation matrix of the RMSD between both trajectories. This is sort of useful for like comparings RMSDs when your system is the same (since only one structure can be used with -s), but perhaps your run conditions are different (diff temperature, pressure, starting velocity, constraints, etc.). This is what the matrix looks like
![plot](https://user-images.githubusercontent.com/91963008/163006902-92ed4d15-fbc3-4dc8-9e84-5a3767298a4c.png)
: